Add ports customisation options for ingress controller service. (#479)
nginxinc/kubernetes-ingress: deployments/helm-chart/README.md

`controller.service.loadBalancerIP` | The static IP address for the load balancer. Requires `controller.service.type` set to `LoadBalancer`. The cloud provider must support this feature.  | None
`controller.service.loadBalancerSourceRanges` | The IP ranges (CIDR) that are allowed to access the load balancer. Requires `controller.service.type` set to `LoadBalancer`. The cloud provider must support this feature. | []
`controller.service.externalIPs` | The list of external IPs for the Ingress controller service. | []

> `controller.service.httpPort.enable` | Enable Http port for the Ingress controller service. | true

Reviewer: (Optional, for consistency)

```suggestion
`controller.service.httpPort.enable` | Enables the HTTP port for the Ingress controller service. | true
```

> `controller.service.httpPort.enable` | Enables the Http port for the Ingress controller service. | true

Reviewer: ```suggestion
`controller.service.httpPort.enable` | Enables the HTTP port for the Ingress controller service. | true
```

> `controller.service.httpPort.port` | Number port for Http service of the Ingress controller service. | 80

Reviewer: (Optional, for consistency)
```suggestion
`controller.service.httpPort.port` | The HTTP port of the Ingress controller service. | 80
```

> `controller.service.httpPort.nodePort` | The custom NodePort for the HTTP port. Requires `controller.service.type` set to `NodePort` and `controller.service.httpPort.enable` set to `true`. | None

Reviewer: ```suggestion
`controller.service.httpPort.nodePort` | The custom NodePort for the HTTP port. Requires `controller.service.type` set to `NodePort`. | ""
```

> `controller.service.httpPort.nodePort` | Custom NodePort number for http. Requires `controller.service.type` set to `NodePort` and `controller.service.httpPort.enable` set to `true`. | None

Reviewer: ```suggestion
`controller.service.httpPort.nodePort` | The custom NodePort for the HTTP port. Requires controller.service.type set to NodePort. | ""
```

> `controller.service.httpsPort.enable` | Enable Https port for the Ingress controller service. | true

Reviewer: (Optional, for consistency)
```suggestion
`controller.service.httpsPort.enable` | Enables the HTTPS port for the Ingress controller service. | true
```

> `controller.service.httpsPort.port` | Number port for Https service of the Ingress controller service. | 443

Reviewer: (Optional, for consistency)
```suggestion
`controller.service.httpsPort.port` | The HTTPS port of the Ingress controller service. | 443
```

> `controller.service.httpsPort.nodePort` | The custom NodePort for HTTPS port. Requires `controller.service.type` set to `NodePort` and `controller.service.httpsPort.enable` set to `true`.  | None

Reviewer: ```suggestion
`controller.service.httpsPort.nodePort` | The custom NodePort for HTTPS port. Requires `controller.service.type` set to `NodePort`.  | ""
```

> `controller.service.httpsPort.nodePort` | Custom NodePort number for https. Requires `controller.service.type` set to `NodePort` and `controller.service.httpsPort.enable` set to `true`.  | None

Reviewer: ```suggestion
`controller.service.httpsPort.nodePort` | The custom NodePort for the HTTPS port. Requires `controller.service.type` set to `NodePort` and `controller.service.httpsPort.enable` set to `true`.  | ""
```
